components/tools/OmeroPy/src/omero/__init__.py

```diff
@@ -104,6 +104,14 @@ def proxy_to_instance(proxy_string, default=None):
     """
     import omero
     parts = proxy_string.split(":")
+    # If a default is provided and the format is Class:ID
```

Comment:
I'm a bit confused where this is coming from. Is something causing incorrect strings to be passed when a default is used? My understanding of the use of the default (e.g. "Fileset") is that:

```
 1         ->  Fileset:1
 Fileset:1 ->  Fileset:1
 Image:1   ->  Image:1
```